.github/workflows/ci-build.yaml

```diff
@@ -505,6 +513,13 @@ jobs:
         run: |
           set -x
           make test-e2e-local
+        continue-on-error: true
+      - name: Upload e2e coverage report
+        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
+        with:
+          name: e2e-code-coverage
+          path: /tmp/coverage/cov*
+        if: ${{ matrix.k3s-version == 'v1.29.1' }}
```

Comment:
i hate this because it requires us to make a change in two places. the matrix and here. i am not sure how to only run the upload once instead of for each k3s version.

Reply:
I wonder if the members of the matrix can be objects instead of string. i.e. 

```
- k3s-version: 'v1.29.1'
  is-latest: true
```

That way we only have to reference `is-latest`.

Reply:
i bet you can!